agent/service.go

```diff
@@ -150,7 +151,13 @@ func (ts *TaskService) doCleanup(taskExecID string) error {
 
 // Create creates a new initial process and container using runc
 func (ts *TaskService) Create(requestCtx context.Context, req *taskAPI.CreateTaskRequest) (_ *taskAPI.CreateTaskResponse, err error) {
-	defer logPanicAndDie(log.G(requestCtx))
+	logger := log.G(requestCtx).WithFields(logrus.Fields{
+		"name":    "Create",
```

Comment:
Open to feedback on the key name here.

Reply:
Did you remove the log panic call?

Reply:
It is line 159 now, basically setting up the logger with fields so if a panic occurs we can see what function and container caused the panic.